includes/discovery/sensors/pre-cache/timos.inc.php

```diff
@@ -11,3 +11,6 @@
  */
 
 $pre_cache['timos_oids'] = snmpwalk_cache_multi_oid($device, 'tmnxDigitalDiagMonitorEntry', [], 'TIMETRA-PORT-MIB', 'timos');
+$pre_cache['timos_lane_oids'] = snmpwalk_cache_multi_oid($device, 'tmnxPortEntry', [], 'TIMETRA-PORT-MIB', 'timos');
```

Comment:
Please put these in the discovery file and not in pre-cache.  They aren't shared across modules at all.  Using pre-cache for data like this makes things slower.